src/SFA.DAS.PAS.Host.WebJob/Web.config

```diff
@@ -8,6 +8,6 @@
     <!--the targetFramework mismatch is valid (https://stackoverflow.com/questions/40170567/what-do-the-targetframework-settings-mean-in-web-config-in-asp-net-mvc)-->
     <!--but should we update the httpRuntime targetFramework?-->
     <compilation debug="true" targetFramework="4.6.2"/>
-    <httpRuntime targetFramework="4.5"/>
+    <httpRuntime targetFramework="4.6.2"/>
```

Comment:
is this an intended change?

Reply:
Yea, we've upgraded the project to 4.6.2, so this is saying we want it to run under 4.6.2